tests/test_views.py

```diff
@@ -0,0 +1,750 @@
+import json
+
+from lxml import objectify
+import pytest
+
+from django.core.urlresolvers import reverse
+from django.http import Http404
+from django.utils import timezone
+
+from premis_event_service import views, models
+from . import factories, AppEventTestXml
+
+
+pytestmark = [
+    pytest.mark.urls('premis_event_service.urls'),
+    pytest.mark.django_db,
+]
+
+
+def test_app_returns_ok(rf):
+    request = rf.get('/')
+    response = views.app(request)
+    assert response.status_code == 200
+
+
+def test_app_response_content_type(rf):
+    request = rf.get('/')
+    response = views.app(request)
+    assert response.get('Content-Type') == 'application/atom+xml'
+
+
+def test_humanEvent_raises_http404(rf):
+    """An Http404 should be raised when the view is unable to locate
+    an Event.
+    """
+    request = rf.get('/')
+    with pytest.raises(Http404):
+        views.humanEvent(request)
+
+
+def test_humanEvent_returns_ok(rf):
+    event = factories.EventFactory.create()
+    request = rf.get('/')
+    response = views.humanEvent(request, event.event_identifier)
+    assert response.status_code == 200
+
+
+def test_json_event_list_returns_ok(rf):
+    factories.EventFactory.create_batch(30)
+    request = rf.get('/')
+    response = views.recent_event_list(request)
+    assert response.status_code == 200
+
+
+def test_json_event_list_context(client):
+    """Test the Events in the response context."""
+    num_events = 30
+    factories.EventFactory.create_batch(num_events)
+    response = client.get(reverse('premis_event_service.views.recent_event_list'))
+
+    context = response.context[-1]
+    assert len(context['entries']) == 10
+    assert context['num_events'] == num_events
+
+
+def test_json_event_list_with_no_events(rf):
+    """Test that the response has a status code 200 when there are no
+    Events in the database.
+    """
+    request = rf.get('/')
+    response = views.recent_event_list(request)
+    assert response.status_code == 200
+
+
+def test_json_agent_returns_ok(rf):
+    agent = factories.AgentFactory.create()
+    request = rf.get('/')
+    response = views.json_agent(request, agent.agent_identifier)
+    assert response.status_code == 200
+    assert response.get('Content-Type') == 'application/json'
```

Comment:
Having both these assertions in the same test seems to diverge from what your first tests looked like. There they were split into separate tests. I don't know that either needs to be changed, but it's something to be aware of.